ffs/overview.md

```diff
@@ -0,0 +1,73 @@
+
+### Completed Proposal
+Sit back, put on your party hat, and celebrate. What once started as an idea is now a reality. Things are now finished and the Monero community benefits from yet another completed proposal. This is what makes Monero special. The community believes in the tech and in its members. We did it. Many more proposals to come.
+
+### A note about Expiration
+In some rare cases, a proposal is abandoned by the individual or team who has agreed to work on it. In these cases the proposal enters a special 'Expired' state, which is somewhere in between Open Tasks and Work in Progress. The funds have already been raised, but a new individual is needed to take on the project. Usually the community has to accept a new person that will work on the project that they feel would be qualified to do so.
+
+## 5. General FFS Guidelines
+*Note: These guidelines are different than the official FFS terms, which are required reading before submitting a proposal. These guidelines are just general ideas for the best way to have a smooth experience with the FFS.*
+
+* Read the Terms.
+* If you are trying to distill community reaction to an idea, it's recommended you come to the appropriate IRC channel for feedback. The people there are usually quite serious about Monero and more than willing to help out. The list of common IRC hangouts can be found [here](https://getmonero.org/community/hangouts)
+* If you are an individual who wants to complete some ideas, it helps if the community knows who you are first. While it's possible for newcomers to propose and complete new ideas through the FFS, it's not common. Build a reputation of service to Monero and its community. That usually means doing some work pro-bono, or otherwise proving your qualities and skills.
+* When your proposal is in the Funding Required stage, it's fine to let people know about by posting in the various [Hangouts](https://getmonero.org/community/hangouts).
+* Be willing to amend the proposal pre-Work in Progress. You may have to make some tweaks in milestones, price, or scope as determined by community feedback. If you're an expert and you feel strongly about keeping something a certain way, make your case in a reasonable manner. The community just wants what's best for Monero, and as the expert, you have to explain to them why your way makes more sense.
+* If you've completed a milestone and ping a core team member for a payout, be patient. They have lives to, and aren't paid to work on Monero. That being said, they realize they forget things and don't take offense if you poke them every once in a while to remind them.
```

Comment:
It is possible for us to, yes, but that line is still presently accurate. Maybe it could be made more clear somehow, essentially to reflect that we aren't compensated _for_ being on the core team, but could be for other things (even FFS management or whatever other "core" things we are doing for free now).